aspnetcore/fundamentals/logging/index.md

```diff
@@ -523,7 +523,7 @@ The order of placeholders, not their names, determines which parameters are used
 ```csharp
 string p1 = "parm1";
 string p2 = "parm2";
-_logger.LogInformation("Parameter values: {p1}, {p2}", p1, p2);
+_logger.LogInformation("Parameter values: {p2}, {p1}", p1, p2);
```

Comment:
I thought these placeholders were filled from left to right in order ... is that not the case?

Reply:
oh ... wait ... you're saying they are out of sequence on purpose in the context of the topic?

Reply:
oh! gotcha ...... sorry .... that whole PR was a giant exercise in find-and-replace. :smile: lol